libs/vertexai/langchain_google_vertexai/_utils.py

```diff
@@ -138,7 +138,7 @@ def _missing_(cls, value: Any) -> "GoogleModelFamily":
             return GoogleModelFamily.CODEY
         if "medlm-medium@latest" in model_name:
             return GoogleModelFamily.GEMINI
-        if "bison" in model_name or "medlm" in model_name:
+        if "bison" in model_name or "medlm" in model_name or "unicon" in model_name:
```

Comment:
Should be `unicorn` here? Could you update, please 🙏

Reply:
Tried to update myself but I don't have the permissions to push.

Reply:
@lkuligin FYI 😄

Reply:
sorry, fixed